src/Uno.UI-Android-hotreload-only.slnf

```diff
@@ -0,0 +1,32 @@
+{
+  "solution": {
```

Comment:
The solution filter changes seem inconsistent. New filters for Android and WASM (with different naming schemes) but not for iOS, changes to existing filters for WASM and iOS but not Android.

Reply:
Fixed!